listMembersOfaJobGroups.groovy

```diff
@@ -0,0 +1,17 @@
+import nectar.plugins.rbac.groups.*
```

Comment:
I'd include a description about what this is about. Something like:

/*
Author: Carlos Rodriguez Lopez
Since: September 2017
Description: It retrieves all the job templates items available in the instance and include them into the restrict list of items to create inside pFolder
Parameters: pFolders
Scope: Cloudbees Jenkins Enterprise
*/